### ISSUE TITLE
Popover causes the browser to crash if window size is changed

### ISSUE THREAD

**Issue by liemdo** [reactstrap/reactstrap#1488]
- components: `popover`
- reactstrap version `8.0.0`

### Steps to reproduce the issue

1. Open the popover in Chrome or Safari (I didn't test with other browsers)
2. Change the window size
3. The browser crashes. I have to force quit the browser.

**Comment by TheSharpieOne** [comment]
I cannot reproduce. Does this happen for you on https://reactstrap.github.io/components/popovers/ ?

**Comment by liemdo** [comment]
You have to resize the window quickly so that part of the popover is out of the window view. The browser crashes right after that.

![reactstrap](https://user-images.githubusercontent.com/989817/57551219-1fc1df80-732e-11e9-9d36-ee58bf59d528.gif)

If you open the Chrome Devtools while the popover is open, you'll get the same issue. 

This issue may be related to this one https://github.com/reactstrap/reactstrap/issues/1482

**Comment by morel333** [comment]
> I cannot reproduce. Does this happen for you on https://reactstrap.github.io/components/popovers/ ?

Open your samples in minimal browser size and click on any popover - it will reproduce.
Looks like ther's an infinite state placement change in PopperContent in handlePlacementChange (placement changes top/bottom -> etc).

**Comment by vitorblz** [comment]
The same happens with tooltips

**Comment by morel333** [comment]
As a hotfix changed
`        <ReactPopper
          referenceElement={this.targetNode}
          modifiers={extendedModifiers}
          placement={placement}
        >`
to
`<ReactPopper
                    referenceElement={this.targetNode}
                    modifiers={extendedModifiers}
                    placement={attrs.placement}
                >`
in PopperContent

**Comment by usman-subhani** [comment]
![Tooltiperror](https://user-images.githubusercontent.com/25919458/58682585-9892e500-838a-11e9-9268-90c5e9831be0.gif)

My browser crashes as well while using tooltips.Its a bit tricky to reproduce but as far as i know it often happens when mouse moves to quickly over the tooltip hover element or mouse hovers over a tooltip element during/after scrolling. I am using uncontrolled tooltips with delays mentioned below.
I am using v8.0.0 of reactstrap.
```
const tooltipDelay = { show: 500, hide: 300 };

const UncontrolledTooltipWithDelay = props => (
  <UncontrolledTooltip delay={tooltipDelay} trigger="hover" {...props} />
);
```

**Comment by Phibedy** [comment]
For me it happens if the tooltip does not fit in the window. As long as the tooltip fits on the page it works fine but if I keep decreasing the window size it fails.
![image](https://user-images.githubusercontent.com/1841954/58751939-637bb500-84a6-11e9-9d86-f1cdf99d44d2.png)

**Comment by YassienW** [comment]
@usman-subhani Would this fix your issue? https://github.com/reactstrap/reactstrap/issues/1482#issuecomment-498785625

**Comment by usman-subhani** [comment]
> @usman-subhani Would this fix your issue? [#1482 (comment)](https://github.com/reactstrap/reactstrap/issues/1482#issuecomment-498785625)

@YassienW Yes downgrading to v7.1.0 of reactstrap resolved this for me. Thanks 👍

**Comment by jesusgp22** [comment]
This issue also happens if the popover is bigger than the current viewport on opening the popover
